### ISSUE TITLE
Getting feature names from ColumnTransformers and Pipelines

### ISSUE THREAD

**Issue by abhi8893** [abhi8893/Titanic-Survival#12]
Look into this 
https://stackoverflow.com/questions/54646709/sklearn-pipeline-get-feature-name-after-onehotencode-in-columntransformer
https://stackoverflow.com/questions/57528350/can-you-consistently-keep-track-of-column-labels-using-sklearns-transformer-api/57534118#57534118
https://stackoverflow.com/questions/53382322/adding-get-feature-names-to-columntransformer-pipeline
https://zablo.net/blog/post/pandas-dataframe-in-scikit-learn-feature-union/index.html
https://github.com/scikit-learn-contrib/sklearn-pandas
https://github.com/jpmml/sklearn2pmml
